docs/sections/links.rst

```diff
@@ -0,0 +1,13 @@
+.. _anaconda: https://anaconda.org/maddenp/condev  
+.. _black: https://black.readthedocs.io/en/stable/ 
+.. _condev: https://github.com/maddenp/condev
+.. _condev demo: https://github.com/maddenp/condev/tree/main/demo
+.. _coverage: https://coverage.readthedocs.io/en/7.2.7/ 
+.. _docformatter: https://docformatter.readthedocs.io/en/stable/ 
+.. _isort: https://pycqa.github.io/isort/
+.. _jq: https://jqlang.github.io/jq/manual/v1.6/ 
```

Comment:
Previous link was the manual for version 1.7. We have 1.6 version in out `meta.yaml`. 

Previous link: https://devdocs.io/jq/